eyeshade/workers/reports.js

```diff
@@ -1377,7 +1421,7 @@ exports.workers = {
       })
       if (!publisher) {
         data.push({
-          publisher: 'TOTAL',
+          note: 'TOTAL IN',
```

Comment:
i suspect that we'll get an ask to change /IN/ to something else, but i'm not sure what.